FlowCrypt/Controllers/Threads/ThreadDetailsDecorator.swift

```diff
@@ -77,3 +69,46 @@ extension AttachmentNode.Input {
         )
     }
 }
+
+private func makeEncryptionBadge(_ input: ThreadDetailsViewController.Input) -> BadgeNode.Input {
+    let icon: String
+    let text: String
+    let color: UIColor
+    switch input.processedMessage?.messageType {
+    case .error:
+        icon = "lock.open"
+        text = "message_decrypt_error".localized
+        color = .red
+    case .encrypted:
+        icon = "lock"
+        text = "message_encrypted".localized
+        color = .main
+    default:
+        icon = "lock.open"
+        text = "message_not_encrypted".localized
+        color = .warningColor
```

Comment:
While we're editting this, please also change this `.warningColor` to `.errorColor` here. Thanks!